### PR TITLE
Cockpit 221

### PR DESCRIPTION
**Pull request by mvollmer**
https://mvollmer.github.io/cockpit-project.github.io/blog/cockpit-221.html

### CODE REVIEW

File: _posts/2020-06-10-cockpit-221.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Instead of disabling the appropriate buttons them when the logged in
+user doesn't have administrative access, they will now simply
+disappear.  This is a general theme that we want to roll out all over
+Cockpit.  The Storage and Network pages have already been changed like
+this in earlier releases.
+
+### Developers: Importing "base1/patternfly.css" is deprecated
+
+This pre-compiled stylesheet will be dropped in the future in favor of
```
Comment (martinpitt) [inline _posts/2020-06-10-cockpit-221.md:34]: I updated the description in https://github.com/cockpit-project/cockpit/pull/14105 a bit.

File: _posts/2020-06-10-cockpit-221.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+header to `same-origin`. Web browsers use this header to prevent other
+web sites from loading individual HTML, JavaScript, or image resources
+from cockpit via `<img>`, `<script>`, or similar tags. This is mostly
+a precaution -- currently there are no known vulnerabilities with
+including resources that way, but as this is not a legitimate use
+case, there is no reason to allow it.
+
+### Accounts: Some buttons are hidden when access is limited
+
+Instead of disabling the appropriate buttons them when the logged in
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply _posts/2020-06-10-cockpit-221.md:27]: Yep.

File: _posts/2020-06-10-cockpit-221.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+Cockpit's web server now sets the
+[Cross-Origin-Resource-Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cross-Origin_Resource_Policy_(CORP))
+header to `same-origin`. Web browsers use this header to prevent other
+web sites from loading individual HTML, JavaScript, or image resources
+from cockpit via `<img>`, `<script>`, or similar tags. This is mostly
+a precaution -- currently there are no known vulnerabilities with
+including resources that way, but as this is not a legitimate use
+case, there is no reason to allow it.
+
+### Accounts: Some buttons are hidden when access is limited
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply _posts/2020-06-10-cockpit-221.md:25]: Good point.

File: _posts/2020-06-10-cockpit-221.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+### Developers: Importing "base1/patternfly.css" is deprecated
+
+This pre-compiled stylesheet will be dropped in the future in favor of
+projects shipping their own CSS. This API is not maintainable, as
+Cockpit cannot offer a PatternFly 3 API forever, and PatternFly 4 also
+changes quickly enough that one style sheet for all projects is not
+robust enough.
+
+The cockpit plugins that are using only [PatternFly
```
[the remark this answers is not in the document]
Comment (mvollmer) [reply _posts/2020-06-10-cockpit-221.md:41]: Fixed everywhere, good catch.

File: _posts/2020-06-10-cockpit-221.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+implemented in this [cockpit-podman
+commit](https://github.com/cockpit-project/cockpit-podman/commit/86fa1ee8b84ff50db0623a453de094b47a5bc2a4).
+
+### Try it out
+
+Cockpit 221 is available now:
+
+ * [For your Linux system](https://cockpit-project.org/running.html)
+ * [Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/221)
+ * [Fedora Rawhide](https://bodhi.fedoraproject.org/updates/FEDORA-2020-db8468a77a)
```
Comment (martinpitt) [inline _posts/2020-06-10-cockpit-221.md:58]: Fedora infra plays along again. I just did the missing builds. Bodhi for F31: https://bodhi.fedoraproject.org/updates/FEDORA-2020-983ae499fa and F32: https://bodhi.fedoraproject.org/updates/FEDORA-2020-9c0127be8c

Comment (martinpitt) [reply _posts/2020-06-10-cockpit-221.md]: I pushed the bodhi URLs to this PR.

**Pull request merged**